proposed/simplecache.md

```diff
@@ -111,26 +111,24 @@ namespace Psr\SimpleCache;
 
 interface CacheInterface
 {
-    const TTL_MINUTE = 60;
-    const TTL_HOUR = 3600;
-    const TTL_DAY = 86400;
```

Comment:
Because they are kinda useless and someone suggested dropping them.. Any dev worth their salt probably knows 60/3600/86400 by heart.

I don't see the rationale of keeping this but dropping the Bridge for example. Note that the bridge is not in the spec, it's just in the package, I get that cache-util was made a separate package as it contains a bunch of code, but in this case it's *one* class with a few lines of code.